uefi/src/mem/aligned_buffer.rs

```diff
@@ -67,6 +79,14 @@ impl AlignedBuffer {
         }
     }
 
+    /// Fill the aligned memory region with data from the given iterator.
+    /// If the given iterator is shorter than the buffer, the remaining area will be left untouched.
+    pub fn copy_from_iter(&mut self, src: impl Iterator<Item = u8>) {
```

Comment:
commit `uefi: Fix unaligned memcpy in ConfigurationString parser`

Could you please explain _why_ and _where_ an error occurred? I can't see it on a first glance. Miri error? runtime error?

Reply:
`[u16]` arrays have to use an alignment of `2`. (The start memory address has to be divisble by `2`).
I illegaly cast a `[u8]`, whose alignment requirement is more lax to a `[Char16]`.

This was a miri error `uefi`'s CI gave me.